desktop/ui/src/main/java/org/datacleaner/cli/CliRunner.java

```diff
@@ -70,7 +73,6 @@
  * Contains the execution logic to run a job from the command line.
  */
 public final class CliRunner implements Closeable {
-
```

Comment:
OK, it really doesn't matter but I am starting to notice that you generally don't like this extra line after the class declaration, but I kind of like it. But it's just a damn visual preference, like space between methods or whatever. So we should just maybe agree on what is best.